bin/metric_ingestion_submit.py

```diff
@@ -147,10 +147,10 @@ def main(args=None):
                 logger.print_and_log(logging.CRITICAL, "\nJob: "+"'"+job_namespace+"' is already running", 1)
             else:
                 logger.print_and_log(logging.INFO, "Everything is ok")
```

Comment:
Great catch! In addition, the logging should be moved out of the FOR loop, above the check call.